staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -864,6 +864,7 @@ type ReplicaSetCondition struct {
 
 // PodSecurityPolicy governs the ability to make requests that affect the Security Context
 // that will be applied to a pod and container.
+// Deprecated: use PodSecurityPolicy from policy/v1beta1 API Group instead.
```

Comment:
omit the version... we don't want to need to update this when it gets promoted in the policy group